### ISSUE TITLE
Nothing in the XEP says an owner is mandatory.

### ISSUE THREAD

**Issue by imaginator** [buddycloud/buddycloud-xep#53]
From a recent comment:

Everyone is a follower+post? Should it have an owner?

**Comment by lloydwatkin** [comment]
Yes, the server (now) enforces at least one owner. The value for 'owner' stored in the node config is just an FYI and has no bearing on who the actual owner is (its more a of 'creator' field).

**Comment by imaginator** [comment]
I'll make sure that this gets rolled into the spec. I need a day when there are not distractions and then I'll actually update the XEP. #most-boring-job-ever!
